backend-services/pyproject.toml

```diff
@@ -0,0 +1,84 @@
+[tool.poetry]
+name = "wallet-backend-services"
+version = "0.1.0"
+description = ""
+authors = ["Nautilus"]
+readme = "README.md"
+packages = [
+         {include = "common", from = "src"},
+         {include = "data_service", from = "src"},
+         {include = "web_asgi", from = "src"},
+         {include = "tests"},
+         ]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+motor = "^3.1.1"
+fastapi = "^0.87.0"
+pydantic = "^1.10.2"
+python-dotenv = "^0.21.0"
+uvicorn = {extras = ["standard"], version = "^0.20.0"}
```

Comment:
Could you write a high level description of these libraries and what they do, also for ruff, black, etc. Two or three paragraphs summarizing how these fit together since it's the first use in our codebase.

Reply:
Definitely, not a problem!